docs/UserGuide.md

```diff
@@ -147,40 +147,55 @@ Format: `sort`
 - If their birthday has already passed, the calculation is based on the number of days until their next birthday next year.
 
 
-### Clearing Realodex : `clearrealodex`
+### Clearing Realodex : `clearRealodex`
 
 Clears all clients in Realodex.
 
-Format: `clearrealodex`
+Format: `clearRealodex`
 
 ### Help : `help`
 
 Generates a pop-up window which is a summarised version of the User Guide.
 
 Format: `help`
 
+### Help (Individual Commands) : `COMMAND help`
+
+Shows the help message for the specified command only.
+
+Format: `COMMAND help`
+
+Examples:
+- `add help` provides the help message for the add command
+- `filter help` provides the help message for the filter command
+
 ### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
-### Saving the data
+### File Data
+
+The file that stores the data of your contacts can be found in the same folder/directory as the Realodex app. (e.g. if you
+have Realodex installed in your Desktop, the data file can be found in your Desktop as well.)
+
+#### Saving Data
 
-Realodex data is saved in the hard disk automatically after any command that modifies it. There is no need to save manually.
+Realodex data is saved in the hard disk as a JSON file automatically after any command that modifies it. There is no need to save manually.
 
-### Editing the data file
+#### Editing the data 
 
 Realodex data is saved automatically as a JSON file `[JAR file location]/data/realodex.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <box type="warning" seamless>
 
 **Caution:**
-If your changes to the data file makes its format invalid, Realodex will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
+If your changes to the data file makes its format invalid, Realodex will discard all data and start with an empty data file at the next run.  Hence, it is recommended to make a backup of the file before editing it.<br>
 Furthermore, certain edits can cause the Realodex to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </box>
 
-### Filtering clients by income and more `[coming in v1.3]`
+### Filtering clients by income and more `[coming soon...]`
 
```

Comment:
Actually might be good to give them a backup option in case the json file does not load, cause we often do this oso. Uty where this goes but can mention clearly, if they wish to have a new copy, delete realodex.json and start `Realodex` again